TribalWars/src/Infrastructure/Repository/PlayerRepository.cs

```diff
@@ -1,20 +1,38 @@
-﻿using ApplicationCore.Interfaces.Repository;
-using Infrastructure.Data;
-using Microsoft.EntityFrameworkCore;
 using System;
 using System.Threading.Tasks;
+using ApplicationCore.Entities;
+using ApplicationCore.Interfaces.Repository;
+using Infrastructure.Data;
+using Microsoft.EntityFrameworkCore;
 
 namespace Infrastructure.Repository
 {
-    public class PlayerRepository : BaseRepository, IPlayerRepository
```

Comment:
All the other Repositories use BaseRepository inheritance, why should this one be different?